components/MainAccordian.tsx

```diff
@@ -70,15 +72,15 @@ const MainAccordian = () => {
                         tabIndex={0}
```

Comment:
@vsauceda-cfa This is not super important, but if you want to avoid rewriting the attributes for each tag, you can simply write them in one object and spread them as many times as you need. `JSX` tags behave like regular JS objects. So instead of

```
                       <a
                        title={activeCopy.href2}
                        tabIndex={0}
                        style={{ display: 'block' }}
                        className={classes.accordDescriptionLink}
                        onClick={pushToSearch}
                      >
```
written multiple times, you could have

```
const anchorProps = { tabIndex : 0, style : { display : 'block' }, className : classes.accordDescriptionLink, onClick : pushToSearch }

                      <a title={activeCopy.href2} {...anchorProps} >
```
and that will be a little cleaner and more concise.